client.yml

```diff
@@ -15,7 +15,7 @@ info:
   license:
     name: Apache 2.0
     url: https://www.apache.org/licenses/LICENSE-2.0.html
```

Comment:
Not a big deal, but we should probably be bumping the minor whenever something new is added